### PR TITLE
Local DynamoDB

### PR DESCRIPTION
**Pull request by jimmyfnugent**
Stand up a local DynamoDB on local server runs. The serverless function automatically connects to it.

### CODE REVIEW

**Comment by jimmyfnugent** [comment]
This is supposed to build on create-bits but I don't think there's a way to use that as a diffbase. Lol

**Comment by jimmyfnugent** [comment]
Aha! https://github.com/hopskipnfall/smashbits/compare/create-bits...dynamo-local?diff=split&name=dynamo-local

File: server/package.json [hunk 1]
```diff
@@ -52,6 +52,8 @@
     "immutable": "^3.8.1",
     "js-string-escape": "^1.0.1",
     "redux": "^3.7.2",
+    "serverless-dynamodb-client": "0.0.2",
+    "serverless-dynamodb-local": "^0.2.28",
```
Comment (hopskipnfall) [inline server/package.json]: Add as a dev dependency? https://github.com/hopskipnfall/jonn-bot/blob/master/package.json#L29

Comment (jimmyfnugent) [reply server/package.json]: Done.

**Pull request merged**